module-2/data-analysis-with-pandas/your-code/main.ipynb

```diff
@@ -1265,7 +1265,7 @@
      "name": "stdout",
```

Comment:
- When you use to_numeric method that way your numeric dtype is floating number which in your case doesn't make sense. You only have integers (int)
- When you have big float numbers, there is a way of adjusting python to display in a proper manner. The method is pd.set_options(), some of the arguments let us to adjust the way we display 'numbers'. Feel free to explore it. The way scientific notation is applied is controlled via pandas' options. 
- Other way of doing the above is applying some kind of transformation on numeric variables to change the scale but not the nature of your data a.k.a. logarithmic transformation. In this case, there is some additional stuff you should do in order to perform your analysis first and show your final insights. 
- I would suggest to explore the kind of data you have before calculating all kind of correlation coef. Take a look on this post, it's pretty clear. You don't have ordinal variables in your dataset so the spearman correlation is not very useful. https://stats.idre.ucla.edu/other/mult-pkg/whatstat/what-is-the-difference-between-categorical-ordinal-and-interval-variables/
- It seems that your data set has some outliers (extreme values) that make quite difficult to get a clear picture in a scatter plot. You can plot these 2 variables removing some of your outliers in order to get a meaningful plot.